server/karaf/features/src/main/resources/features.xml

```diff
@@ -54,8 +54,6 @@
         <bundle>mvn:org.apache.james/apache-mailet-api/${mailet.version}</bundle>
         <bundle>mvn:org.apache.james/apache-mailet-base/${mailet.version}</bundle>
         <bundle>mvn:commons-io/commons-io/${commons-io.version}</bundle>
-        <bundle>mvn:org.apache.geronimo.javamail/geronimo-javamail_1.4_mail/${geronimo-javamail.version}</bundle>
-        <bundle>mvn:org.apache.geronimo.specs/geronimo-activation_1.1_spec/${geronimo-activation-spec.version}</bundle>
```

Comment:
You didn't revert all changes for karaf... Any reason?

Reply:
Just missed them...